packer.mk

```diff
@@ -99,6 +99,9 @@ test-building-vhd: az-login
 scanning-vhd: az-login
 	@./vhdbuilder/packer/vhd-scanning.sh
 
+test-scan-and-cleanup:
+	@$(MAKE) -f packer.mk cleanup test-building-vhd scanning-vhd -j3 --output-sync=target
```

Comment:
the problem with this approach is that is the scan fails and we need to rerun we will have to rerun the all of the test. same with the cleanup, we will need to rerun all of the test and scan/puplish to rerun the cleanup part

Reply:
Yes good point. I will look for a way to only re-run failed targets.